### ISSUE TITLE
Allow multiple avoids and dones per day

### ISSUE THREAD

**Issue by phanirithvij** [Crazy-Marvin/ToDont#124]
#### :warning: Is your feature request related to a problem? Please describe

No

#### :bulb: Describe the solution you'd like

I have a habit of stress nails biting for example, 
I won't do it only once a day, so it makes sense to allow more than one avoid or done for a single habit for a day.

I think this will be better with no frequency restrictions.

#### :bar_chart: Describe alternatives you've considered

Add a counter per habit tile in the Avoided and Done tabs.

#### :raising_hand: Do you want to develop this feature yourself?

- [ ] Yes
- [x] No

**Comment by CrazyMarvin** [comment]
I like your idea.
@arafaatqureshi, do you think this will be difficult to implement?

**Comment by arafaatqureshi** [comment]
@CrazyMarvin yes it is, but not impossible :) will look into it. for that we have to change the whole database structure.

**Comment by teslahitchhiker42** [comment]
So it is possible to add multiple avoids and dones for a habit in one day but I think this may be a bug.

But this is how to do it:

Add a habit to avoided or done.
Go to the avoided or done tab.
Click on the habit you just added. The dialogue for adding notes will come up.
Click update (whether or not you add notes). When you click update, the dialogue closes. But then the date format on the task changes from 2022-10-21 format to 21-Oct-2022 format.
Go back to the habits tab.
Add the same habit to avoided or done (whichever you added it to previously). It will add the habit to avoided or done a second time, but with the original date format (2022-10-21).
You can repeat this all over again for the avoid or done you just added and add yet another one, and I think this process can be repeated indefinitely to add infinite avoids or dones for any habit in one day.

I'm not sure what this does to your log tho.

**Comment by CrazyMarvin** [comment]
@theftzoku is working on this.

**Comment by CrazyMarvin** [comment]
The feature is live with `v3.0.0` thanks to @theftzoku.
